components/brave_wallet_ui/page/screens/send/send/send.tsx

```diff
@@ -247,7 +276,10 @@ export const Send = (props: Props) => {
     data: spotPriceRegistry
   } = useGetTokenSpotPricesQuery(
     tokenPriceIds.length ? { ids: tokenPriceIds } : skipToken,
-    querySubscriptionOptions60s
+    {
+      ...querySubscriptionOptions60s,
+      skip: isLoadingBalances
+    }
```

Comment:
can we can use `skipToken` instead of the skip option?
```suggestion
    !isLoadingBalances && tokenPriceIds.length ? { ids: tokenPriceIds } : skipToken,
    querySubscriptionOptions60s
```